CMakeLists.txt

```diff
@@ -0,0 +1,176 @@
+option(Slicer_BUILD_PARAMETERSERIALIZER_SUPPORT "Build application with parameter serializer support" OFF)
+option(Slicer_USE_PYTHONQT                      "Build application with Python support"               ON)
+option(Slicer_USE_QtTesting                     "Build application with QtTesting support"            OFF)
+option(Slicer_USE_SimpleITK                     "Build application with SimpleITK support"            OFF)
+
+option(Slicer_BUILD_BRAINSTOOLS                 "Build application with BRAINSTools module"           OFF)
+option(Slicer_BUILD_DataStore                   "Build application with DataStore module"             OFF)
+option(Slicer_BUILD_CompareVolumes              "Build application with ChangeTrackerPy module"       OFF)
+option(Slicer_BUILD_LandmarkRegistration        "Build application with LandmarkRegistration module"  OFF)
+option(Slicer_BUILD_SurfaceToolbox              "Build application with SurfaceToolbox module"        OFF)
+
+# Enable Slicer built-in modules
+set(Slicer_CLIMODULES_ENABLED
+  ResampleDTIVolume             # Needed by ResampleScalarVectorDWIVolume
+  ResampleScalarVectorDWIVolume # Depends on DiffusionApplications, needed by CropVolume
+  )
+set(Slicer_QTLOADABLEMODULES_ENABLED
+  )
+set(Slicer_QTSCRIPTEDMODULES_ENABLED
+  )
+
+# Disable Slicer built-in modules
+set(Slicer_CLIMODULES_DISABLED
+  )
+set(Slicer_QTLOADABLEMODULES_DISABLED
+  SceneViews
+  SlicerWelcome
+  ViewControllers
+  )
+set(Slicer_QTSCRIPTEDMODULES_DISABLED
+  DataProbe
+  DMRIInstall
+  Endoscopy
+  LabelStatistics
+  PerformanceTests
+  # SampleData
+  VectorToScalarVolume
+  )
+
+# Enable/Disable Slicer custom modules: To create a new module, use the SlicerExtensionWizard.
+set(Slicer_EXTENSION_SOURCE_DIRS
+  #${OpenLIFU_SOURCE_DIR}/Modules/CLI/MyCLIModule
+  #${OpenLIFU_SOURCE_DIR}/Modules/Loadable/MyLoadableModule
+  ${OpenLIFU_SOURCE_DIR}/Modules/Scripted/Home
+  )
+
+# Add remote extension source directories
+
+# SlicerCustomAppUtilities
+set(extension_name "SlicerCustomAppUtilities")
+set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
+FetchContent_Populate(${extension_name}
+  SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
+  GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/KitwareMedical/SlicerCustomAppUtilities.git
+  GIT_TAG        1d984a2c9143e2617ff1ffa9d86c51e07dc6321e
+  GIT_PROGRESS   1
+  QUIET
+  )
+message(STATUS "Remote - ${extension_name} [OK]")
+list(APPEND Slicer_EXTENSION_SOURCE_DIRS ${${extension_name}_SOURCE_DIR}/Modules/Scripted/SlicerCustomAppUtilities)
+
+# Bundle the SlicerOpenLIFU extension
+set(extension_name "SlicerOpenLIFU")
+set(${extension_name}_SOURCE_DIR "${CMAKE_BINARY_DIR}/${extension_name}")
+FetchContent_Populate(${extension_name}
+ SOURCE_DIR     ${${extension_name}_SOURCE_DIR}
+ GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/OpenwaterHealth/SlicerOpenLIFU.git
+ GIT_TAG        0ea13d4fd39937059085f51fd14f60e6eca81397
```

Comment:
This tag may have to change depending on the fate of https://github.com/OpenwaterHealth/SlicerOpenLIFU/pull/2